src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss

```diff
@@ -0,0 +1,63 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+.card-wrapper {
+    max-width: 320px;
+    min-width: 220px;
+}
+
+$dark-card: igx-card-theme(
+    $background:#fd6bd93a,
```

Comment:
We can format the colors to be consistent with the rest of the topics, for example:
**$background: #FD6BD93A** (space + uppercase)

Reply:
Done